lib/websocket_server.rb

```diff
@@ -35,6 +35,9 @@ def call(env)
       return proxy.cleanup if proxy.error
       proxy.start
 
+      # Release the connection because one SPICE console can open multiple TCP connections
+      ActiveRecord::Base.connection.close
```

Comment:
We use `ActiveRecord::Base.connection_pool.release_connection` in many other places.

```
git grep -n "connection_pool.release_connection"
app/models/miq_worker.rb:475:    ActiveRecord::Base.connection_pool.release_connection if ActiveRecord::Base.connected?
config/application.rb:122:      ActiveRecord::Base.connection_pool.release_connection
lib/miq_automation_engine/engine/miq_ae_method.rb:83:      ActiveRecord::Base.connection_pool.release_connection
lib/miq_automation_engine/engine/miq_ae_method.rb:183:      ActiveRecord::Base.connection_pool.release_connection
lib/miq_automation_engine/engine/miq_ae_service.rb:114:      ActiveRecord::Base.connection_pool.release_connection
lib/miq_automation_engine/engine/miq_ae_service_model_base.rb:288:      ActiveRecord::Base.connection_pool.release_connection rescue nil
lib/miq_automation_engine/service_models/miq_ae_service_methods.rb:161:      ActiveRecord::Base.connection_pool.release_connection rescue nil
```